db/migrate/20210222102503_create_users_settings.rb

```diff
@@ -0,0 +1,25 @@
+class CreateUsersSettings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users_settings do |t|
+      t.references :user, foreign_key: true, null: false
+      t.string "config_font", default: "default", null: false
+      t.string "config_navbar", default: "default", null: false
+      t.string "config_theme", default: "default", null: false
+      t.boolean "display_announcements", default: true, null: false
+      t.boolean "display_sponsors", default: true, null: false
+      t.string "editor_version", default: "v1"
```

Comment:
Maybe editor version should be null: false?

Reply:
I think so, there's no concept of "no editor version".